.github/workflows/codeql-analysis.yml

```diff
@@ -13,12 +13,20 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ master, cluster ]
+    branches: [master, cluster]
+    paths-ignore:
+      - "**/*.md"
+      - "**/*.txt"
```

Comment:
Readme.md is missing actually *.md

Reply:
yeah. sorry this one is WIP however i cant mark as a draft because i'm testing PR workflow. but thanks for catch